Source/towners.cpp

```diff
@@ -233,6 +234,21 @@ void InitCows(Towner &towner, const TownerData &townerData)
 	dMonster[offset.x][offset.y] = -cowId;
 }
 
+void InitBird(Towner& towner, const TownerData& townerData)
+{
+	towner._tAnimWidth = 1;
+	static const uint8_t AnimOrder[] = {
+		// clang-format off
+		1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 
+		// clang-format on
+	};
+	towner.animOrder = AnimOrder;
+	towner.animOrderSize = 10;
+	towner.data = LoadFileInMemFromDisk(R"(D:\development\tmp\pngs2cel\new_bird.cel)");

```

Comment:
Add it to `Packaging/resources/assets/data` in the repo and to `CMake/Assets.cmake`, and then use `LoadFileInMem`

Reply:
Cheers, sounds like a much more reasonable solution.